### DISCUSSION TITLE
GitHub Student  Developer Pack

### DISCUSSION THREAD

**Discussion by rootdash** [community/community#131929] [GitHub Education] [answered]
### Hi everyone,

I'm trying to apply GitHub Student Developer Pack, anytime when trying to submit. It goes back to Application form. I've tried on desktop device and phone device. It still wont submitted and always back to `https://education.github.com/discount_requests/******/additional_information` after hit `Process my application` button.

**Comment by Aditijainnn** [comment, marked as answer]
> ### Hi everyone,
> 
> I'm trying to apply GitHub Student Developer Pack, anytime when trying to submit. It goes back to Application form. I've tried on desktop device and phone device. It still wont submitted and always back to `https://education.github.com/discount_requests/******/additional_information` after hit `Process my application` button.

~check if your school is recognized by GitHub Check to see if your school is in [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). If not, then you may need to contact GitHub Support for additional support.
  ~  Use up to date documents/information. Provide the most up to date information and documents otherwise you might be rejected if it is too outdated.
  ~  Ensure your GitHub profile matches. Make sure all information such as your name or location both match your profile and documents.
  ~ Make sure your school email is linked. You should have your school email verified in your GitHub account.
   ~ Don't use a VPN. GitHub may detect you using a VPN and automatically deny you access. Try applying without any VPN.


You can read "[Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)" and the "[Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)" discussion for more information about Global Campus rejections.

As a reminder you do not need to have a Student ID to get the GitHub Student Developer Pack, for more information see "[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)"

If none of this helps, you can contact GitHub for support at [Get help with GitHub Education](https://support.github.com/contact/education).

I hope this helps you with your question. Once resolved, you can mark this comment as the answer.

**Comment by faelperetta** [comment]
@rootdash, did you manage to submit the application?

**Reply by rootdash** [reply]
Yes, I successfully submitted the application. All you need to do is come to the city where your campus is located, then submit the application. GitHub will detect your connection IP according to the city where your campus is located. Don't use a VPN, it doesn't work. And the application display will be different if you are already in the city where your campus is located.

**Comment by emrsc1** [comment]
nice finally you solved it

**Comment by codewitheren** [comment]
I need a Github student developer pack gift. My application was approved but no gifts

**Comment by Surajasanka** [comment]


**Reply by codewitheren** [reply]
What does this mean? Why did you write this?

**Comment by Sameer-070** [comment]
Hey There! I have some problem with reverification, I have been trying to reverify my student benefits but when I click last "continue" button nothing happens

**Comment by MARafey** [comment]
Hello, I have tried to reapply for my student package multiple times, but all my requests have been rejected. I have used my transcript available on my university portal and my student ID card multiple times, but they still have not been accepted.

**Reply by MARafey** [reply]
![image](https://github.com/user-attachments/assets/87648c24-f5e0-4fb2-abfa-33f2aebf2be7)
![image](https://github.com/user-attachments/assets/01802ac7-104b-4ef2-adb3-bc970ce4bc6c)
